pan/functions.pan

```diff
@@ -44,85 +26,80 @@ function push = {
         error("push can only be applied to a list");
     };
 
-    # Merge the arguments into the given array.  Neither the
-    # first/next or merge functions can be used because the
-    # ARGV array cannot be directly referenced.
+    # Cannot use merge or refernce ARGV directly
     i = 0;
     while (i<ARGC) {
         v[length(v)] = ARGV[i];
```

Comment:
I think this can be replaced with append(ARGV[i]) ? That's what I deployed to our code base internally recently.